openapi.yaml

```diff
@@ -49,156 +95,494 @@ paths:
-    maximum: 500
-    minimum: 1
+  ownerId:
+    name: ownerId
+    in: header
+    description: ID of park owner
+    required: true
+    type: string
+  ownerIdPath:
+    in: path
+    name: ownerId
+    description: ID of owner
+    type: string
+    required: true
+  ownerPassword:
+    name: ownerPassword
+    in: header
+    description: password for park owner
+    required: true
+    type: string
+  parkIdPath:
+    name: parkId
+    in: path
+    description: Unique Park ID
+    required: true
+    type: string
+  
+
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
+        example: request_url_here
         format: url
         description: Self-link of current resource
-  PaginationLinks:
+  OwnerResultPrivate:
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
-        type: string
-        format: url
-        description: The previous page of data
-      next:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/OwnerResourcePrivate'
+  OwnerResultPublic:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/OwnerResourcePublic'
+  OwnersResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/OwnerResourcePublic'
+  OwnerResourcePublic:
+    properties:
+      id:
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
+        example: 0001
+        description: A unique ID of a park owner
+      attributes:
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  OwnerResourcePrivate:
     properties:
       id:
         type: string
-        description: A unique ID of pet
-      type:
+        example: 0001
+        description: A unique ID of a park owner
+      attributes:
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+          ownerPassword:
+            type: string
+            description: Password
+            example: password1234
+  ParkResource:
```

Comment:
For this object and nested objects within this object, could you mark all fields as required? This will specify that all fields should always be returned in the response body